e2e-tests/gef/cypress/e2e/maker/return-to-maker-mia.spec.js

```diff
@@ -108,7 +108,8 @@ context('Return to Maker as MIA', () => {
       applicationDetails.facilitySummaryListTable(0).hasBeenIssuedAction().contains('Change');
       applicationDetails.facilitySummaryListTable(0).monthsOfCoverAction().contains('Change');
       if (facilityEndDateEnabled) {
-        applicationDetails.facilitySummaryListTable(0).isUsingFacilityEndDateAction().contains('Add');
+        applicationDetails.facilitySummaryListTable(0).isUsingFacilityEndDateAction().contains('Change');
+        applicationDetails.facilitySummaryListTable(0).facilityEndDateAction().contains('Change');
```

Comment:
How come the change here?

Reply:
I've updated the mock data so they now contain FED values - the `Add` was here because this data wasnt being updated in the database